plugin/pulp/plugin/importer.py

```diff
@@ -0,0 +1,18 @@
+from pulp.platform.models import RepositoryImporter
+
+
+class RepositoryImporter(RepositoryImporter):
+    """
+    The base RepositoryImporter object
+    """
+
+    def sync(self):
+        """
+        Perform a sync
```

Comment:
@mhrivnak The new docstrings are pretty verbose, but they don't specifically mention this. Would you give them a read through and leave some comments?